node/src/accountant/scanners/mod.rs

```diff
@@ -905,39 +924,62 @@ impl ReceivableScanner {
         }
     }
 
-    fn handle_new_received_payments(&mut self, msg: &ReceivedPayments, logger: &Logger) {
-        let mut txn = self
-            .receivable_dao
-            .as_mut()
-            .more_money_received(msg.timestamp, &msg.payments);
-
-        let new_start_block = msg.new_start_block;
-        match self
-            .persistent_configuration
-            .set_start_block_from_txn(new_start_block, &mut txn)
-        {
-            Ok(()) => (),
-            Err(e) => panic!(
-                "Attempt to set new start block to {} failed due to: {:?}",
-                new_start_block, e
-            ),
-        }
+    fn handle_new_received_payments(
```

Comment:
Even the name alone is a problem a bit. Because it makes you believe some payments were received while you can easily find out soon after you enter, that none was detected.